pythonforandroid/recipes/pycrypto/__init__.py

```diff
@@ -15,7 +15,7 @@ class PyCryptoRecipe(CompiledComponentsPythonRecipe):
     call_hostpython_via_targetpython = False
     patches = ['add_length.patch']
 
-    def get_recipe_env(self, arch=None, clang=True):
+    def get_recipe_env(self, arch=None):
```

Comment:
This change causes the red `X` in travis...I modified the recipe locally and it builded and worked fine at runtime... but even with those changes travis still fails. So no changes applied here to fix this recipe...@AndreMiras, maybe we should add `pycrypto` to BROKEN RECIPES in this pr?

Also see: https://travis-ci.org/kivy/python-for-android/jobs/495925991#L5227-L5247

Reply:
Yes please do so then :+1: